app/angular/package.json

```diff
@@ -120,7 +120,7 @@
     "@nrwl/workspace": ">=11.1.0",
     "rxjs": "^6.0.0 || ^7.4.0",
     "typescript": "^3.4.0 || >=4.0.0",
-    "zone.js": "^0.8.29 || ^0.9.0 || ^0.10.0 || ^0.11.0"
+    "zone.js": "^0.8.29 || >= 0.9.0 < 1.0.0"
```

Comment:
`^0.8.29 || >= 0.9.0 < 1.0.0` can be replaced by `>=0.8.29 < 1.0.0`, but wants to be consistent with Storybook version 7